src/style.scss

```diff
@@ -1,5 +1,10 @@
 * {
   outline: none;
+  -ms-overflow-style: none !important; /* IE and Edge */
+  scrollbar-width: none !important; /* Firefox */
+  &::-webkit-scrollbar {
+    display: none !important;
+  }
```

Comment:
ok, added in https://github.com/OlympusDAO/olympus-frontend/commit/d62e87a8afc668d32b4cb1c0edbbfce8335287ab